templates/mainapp/layout.html

```diff
@@ -212,9 +218,8 @@ <h4 class="modal-title" id="booktitle">Basic Modal</h4>
             $('#booktitle').text(bookName);
             $('#bookauthors').text(authors);
             $('#bookimage').attr('src', imageurl);
-            $("#bookrating").rateYo({
-                rating: averagerating,
-                readOnly: true
+            $(".rateYo").rateYo({
```

Comment:
You cannot set by using the class, are `rateYo` elements will be set to this property if you do this

Reply:
Forgot to change. Changed in new Commit